pkg/util/containers/metrics/provider/metacollector.go

```diff
@@ -13,9 +13,10 @@ import (
 
 // MetaCollector is a special collector that uses all available collectors, by priority order.
 type metaCollector struct {
-	lock                         sync.RWMutex
-	selfContainerIDcollectors    []CollectorRef[SelfContainerIDRetriever]
-	containerIDFromPIDcollectors []CollectorRef[ContainerIDForPIDRetriever]
+	lock                           sync.RWMutex
+	selfContainerIDcollectors      []CollectorRef[SelfContainerIDRetriever]
+	containerIDFromPIDcollectors   []CollectorRef[ContainerIDForPIDRetriever]
+	containerIDFromInodecollectors []CollectorRef[ContainerIDForInodeRetriever]
```

Comment:
```suggestion
	containerIDFromInodeCollectors []CollectorRef[ContainerIDForInodeRetriever]
```
I think we should respect camelcase format even though it is not fully respected above

Reply:
Got it, done 👍